src/constants.js

```diff
@@ -55,6 +55,78 @@ export const CUSTOM_FILTERS = {
                 }
             ]
         }
+    },
+    os: {
+        label: 'Operating system',
+        type: 'checkbox',
+        filterValues: {
+            items: [
+                {
+                    label: 'RHEL 7.0',
+                    value: 'RHEL 7.0'
+                },
+                {
+                    label: 'RHEL 7.1',
+                    value: 'RHEL 7.1'
+                },
+                {
+                    label: 'RHEL 7.2',
+                    value: 'RHEL 7.2'
+                },
+                {
+                    label: 'RHEL 7.3',
+                    value: 'RHEL 7.3'
+                },
+                {
+                    label: 'RHEL 7.4',
+                    value: 'RHEL 7.4'
+                },
+                {
+                    label: 'RHEL 7.5',
+                    value: 'RHEL 7.5'
+                },
+                {
+                    label: 'RHEL 7.6',
+                    value: 'RHEL 7.6'
+                },
+                {
+                    label: 'RHEL 7.7',
+                    value: 'RHEL 7.7'
+                },
+                {
+                    label: 'RHEL 7.8',
+                    value: 'RHEL 7.8'
+                },
+                {
+                    label: 'RHEL 7.9',
+                    value: 'RHEL 7.9'
+                },
+                {
+                    label: 'RHEL 8.0',
+                    value: 'RHEL 8.0'
+                },
+                {
+                    label: 'RHEL 8.1',
+                    value: 'RHEL 8.1'
+                },
+                {
+                    label: 'RHEL 8.2',
+                    value: '8.2'
```

Comment:
@apuntamb I think you missed `RHEL` here.

Reply:
also we need to make sure the value here is same as what backend will accept for OS filter value. 💯

Reply:
Yeah, I have recently made changes in the [backend](https://github.com/RedHatInsights/ros-backend/pull/205/files#diff-a8cad03f08edde21a37e001d0e4efbfb39b3fe06591c7d7ade036dbf91705925R25) and according to those, the values for all should be similar to `8.2`. Will update the same here.

Reply:
@apuntamb Thank you for updating this 💯